rules/downgrade-php70/tests/Rector/FunctionLike/DowngradeTypeParamDeclarationRector/Fixture/array.php.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rector\DowngradePhp70\Tests\Rector\FunctionLike\DowngradeTypeParamDeclarationRector\Fixture;
+
+class ArrayFunctionParam
+{
+    public function run(array $value)
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\DowngradePhp70\Tests\Rector\FunctionLike\DowngradeTypeParamDeclarationRector\Fixture;
+
+class ArrayFunctionParam
+{
+    /**
+     * @param mixed[] $value
+     */
+    public function run($value)
```

Comment:
The `array` type should remain here:

It works on `@param` since PHP 5.3:
https://3v4l.org/haRck

![image](https://user-images.githubusercontent.com/924196/101349835-332c8800-388e-11eb-96d4-9b16f352899d.png)

Reply:
In some other issue, we decided to use `mixed[]` instead of `array` to avoid complaints by PHPStan: https://github.com/rectorphp/rector/issues/4195#issuecomment-730049526. It seems this logic is being applied everywhere. Do you want to change that?

Reply:
I don't talk about docblock. I mean the exact line:

```diff
-public function run($value)
+public function run(array $value)
```

Reply:
ahhhh ok got it. I'll fix it. I can see can receive `array` and also `callable`

Reply:
Fixed